core/modules/czar/MessageTable.cc

```diff
@@ -0,0 +1,124 @@
+/*
+ * LSST Data Management System
+ * Copyright 2015 AURA/LSST.
+ *
+ * This product includes software developed by the
+ * LSST Project (http://www.lsst.org/).
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the LSST License Statement and
+ * the GNU General Public License along with this program.  If not,
+ * see <https://www.lsstcorp.org/LegalNotices/>.
+ */
+
+// Class header
+#include "czar/MessageTable.h"
+
+// System headers
+
+// Third-party headers
+#include "boost/format.hpp"
+
+// LSST headers
+#include "lsst/log/Log.h"
+
+// Qserv headers
+#include "ccontrol/ConfigMap.h"
+#include "ccontrol/UserQuery.h"
+#include "czar/CzarErrors.h"
+#include "sql/SqlConnection.h"
+#include "qdisp/MessageStore.h"
+
+namespace {
+
+LOG_LOGGER _log = LOG_GET("lsst.qserv.czar.MessageTable");
+
+
+std::string const createAndLockTmpl("CREATE TABLE IF NOT EXISTS %1% "
+    "(chunkId INT, code SMALLINT, message CHAR(255), "
+    "severity ENUM ('INFO', 'ERROR'), timeStamp FLOAT)"
+    "ENGINE=MEMORY; LOCK TABLES %1% WRITE;");
+
+std::string const writeTmpl("INSERT INTO %1% (chunkId, code, message, severity, timeStamp) "
+    "VALUES (%2%, %3%, '%4%', '%5%', %6%)");
+
+std::string const unlockTmpl("UNLOCK TABLES");
```

Comment:
Mysql can only unlock all locked table at once, there is no syntax to unlock one table. I'll add comment.